<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="16950" started-at="2021-04-17T19:02:08Z" finished-at="2021-04-17T19:02:25Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="16950" started-at="2021-04-17T19:02:08Z" finished-at="2021-04-17T19:02:25Z">
      <class name="com.testcases.CaptureValueTest">
        <test-method status="PASS" signature="setup()[pri:0, instance:com.testcases.CaptureValueTest@18be83e4]" name="setup" is-config="true" duration-ms="15677" started-at="2021-04-18T00:32:08Z" finished-at="2021-04-18T00:32:24Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="performTest()[pri:0, instance:com.testcases.CaptureValueTest@18be83e4]" name="performTest" duration-ms="288" started-at="2021-04-18T00:32:24Z" finished-at="2021-04-18T00:32:24Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Job search, venture investing & new tech products | AngelList] but found [Remote Job and Startup Job Search | AngelList Talent]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Job search, venture investing & new tech products | AngelList] but found [Remote Job and Startup Job Search | AngelList Talent]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:189)
	at org.testng.Assert.assertEquals(Assert.java:199)
	at com.testcases.CaptureValueTest.performTest(CaptureValueTest.java:33)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- performTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.testcases.CaptureValueTest@18be83e4]" name="tearDown" is-config="true" duration-ms="954" started-at="2021-04-18T00:32:24Z" finished-at="2021-04-18T00:32:25Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.testcases.CaptureValueTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
